src/backend/marsha/core/models/video.py

```diff
@@ -402,4 +402,72 @@ def get_source_s3_key(self, stamp=None):
 
         """
         stamp = stamp or to_timestamp(self.uploaded_on)
-        return f"{self.video.pk}/thumbnail/{self.pk}/{stamp}"
+        return "{video!s}/thumbnail/{pk!s}/{stamp:s}".format(
```

Comment:
Why did you change this ? We don't use string.format function any more.

Reply:
I didn't change it, it's out of my scope, it's a merge issue that I didn't see